agent/README.md

```diff
@@ -123,6 +123,7 @@ echo `git rev-parse --abbrev-ref HEAD`:`git --no-pager log -n1 --format="%ct000"
   section above)
 - `teamscale-git-properties-jar` As an alternative to `teamscale-commit` the agent accepts values supplied via 
   a `git.properties` file generated with [the corresponding Maven or Gradle plugin][git-properties-spring] and stored in a jar/war/ear/...
```

Comment:
I think we should also document the expected format for `git.properties` in case you either want to check whether the plugin is configured correctly and produces the expected format or if you don't use maven, but still want to use the `git.properties`.